modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7559,6 +7560,67 @@ public void testPatchObjectEntryWithTaxonomyCategories() throws Exception {
 			JSONCompareMode.NON_EXTENSIBLE);
 	}
 
+	@Test
+	@TestInfo("LPD-53245")
+	public void testPatchPostPutCustomObjectEntry() throws Exception {
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
I don't know if testing patch here makes sense.

I miss other values for the `PUT`, especially the relationship field (r_<relationshipsName>_<relatedObjectName>Id) but also others such as String, Long, etc, as now we are updating all those fields to the "null" value if they are not sent

What do you think about testing only the PUT method by:

creating with all the possible values

and then, call the PUT without any value

validates everything is by default

and then, call the PUT with all the values

values everything is as expected